projects/ngx-captcha-lib/src/lib/components/base-recaptcha.component.ts

```diff
@@ -150,7 +150,10 @@ export abstract class BaseReCaptchaComponent implements OnChanges, ControlValueA
     ) { }
 
     ngAfterViewInit() {
-        this.control = this.injector.get<NgControl>(NgControl).control;
+        try {
+            this.control = this.injector.get<NgControl>(NgControl).control;
+        }
+        catch {}
```

Comment:
Can you please rethrow original error? I can merge afterwards ;)

Reply:
Great, thanks! The commit I made was just a quick and dirty fix for my project from last summer, but it looks like there's a better way to handle this with no errors, so I just pushed that instead.